sdk/identity/identity/README.md

```diff
@@ -211,6 +219,8 @@ Currently, the following client libraries support authenticating with `TokenCred
 - [@azure/storage-blob](https://npmjs.com/package/@azure/storage-blob)
 - [@azure/storage-queue](https://npmjs.com/package/@azure/storage-queue)
 
+If you need to authenticate to a different service, use the [@azure/ms-rest-js](https://www.npmjs.com/package/@azure/ms-rest-js) client library.
```

Comment:
ms-rest-js is not used for auth, the alternatives would be to use ms-rest-nodeauth for node applications and ms-rest-browserauth for browser applications.

The list above is not complete. We have about 10 more packages that now work with Azure Identity.

Moreover, with the code generator updated as described in https://github.com/Azure/autorest.typescript/issues/815, the Azure Identity library will be usable with all auto generated packages. So, I believe this clarification would not be needed

Reply:
We agreed to undo change to line 222 and remove package list right above it.

Reply:
This is done in #13113